modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java

```diff
@@ -384,8 +354,8 @@ private static void printHelp() {
                 "-s, --update-strategy strategy",
                 "Specifies the update strategy.",
                 "Available strategies:",
-                "- update: (default) new objects will be added to the APISpark's descriptor/connector, primitive fields of existing objects will be updated. Nothing will be deleted.",
-                "- replace: deletes all the information in the descriptor/connector on APISpark's and fills it again with introspected definition.");
+                "- update: (default) new objects will be added to the Restlet Cloud's descriptor/connector, primitive fields of existing objects will be updated. Nothing will be deleted.",
```

Comment:
`new objects will be added to the Restlet Cloud's connector`

Reply:
Bull's eye!